achyut/src/main/java/com/lftechnology/runner/TaskSwitcher.java

```diff
@@ -0,0 +1,42 @@
+package com.lftechnology.runner;
+
+import com.lftechnology.com.programs.MultiplierProgram;
+import com.lftechnology.com.programs.PalindromeProgram;
+import com.lftechnology.com.programs.SalutationProgram;
+
+public class TaskSwitcher {
+
+    protected int chosenOption;
+
+    public TaskSwitcher(int option) {
+        this.chosenOption = option;
+    }
+
+    public void switchAndRun() {
+        Menu menu = new Menu();
+        switch (this.chosenOption) {
+        case 1:
+            SalutationProgram salutationProgram = new SalutationProgram();
+            salutationProgram.run();
+            break;
+        case 2:
+            MultiplierProgram multiplierProgram = new MultiplierProgram();
+            multiplierProgram.run();
+            break;
+        case 3:
+            PalindromeProgram palindromeProgram = new PalindromeProgram();
+            palindromeProgram.run();
+            break;
+        case 0:
+            System.out.println("You are exiting the program now...");
+            System.out.println("Exited");
+            System.exit(0);
+        default:
+            System.out.println("Invalid Input");
+            System.out.println("\n");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)